### PR TITLE
Run Black Formatting on Ocient DB Engine Spec

### PR DESCRIPTION
**Pull request by alexclavel-ocient**
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/db_engine_specs/ocient.py [hunk 1]
```diff
@@ -152,67 +156,71 @@ def _find_columns_to_sanitize(cursor: Any) -> List[PlacedSanitizeFunc]:
...
-    custom_errors : Dict[Pattern[str], Tuple[str, SupersetErrorType, Dict[str, Any]]] = {
+    custom_errors: Dict[Pattern[str], Tuple[str, SupersetErrorType, Dict[str, Any]]] = {
         CONNECTION_INVALID_USERNAME_REGEX: (
             __('The username "%(username)s" does not exist.'),
             SupersetErrorType.CONNECTION_INVALID_USERNAME_ERROR,
             {},
         ),
         CONNECTION_INVALID_PASSWORD_REGEX: (
-            __('The user/password combination is not valid (Incorrect password for user).'),
+            __(
```
Comment (jwilliams-ocient) [inline superset/db_engine_specs/ocient.py:180]: is `__` a function?

Comment (jwilliams-ocient) [reply superset/db_engine_specs/ocient.py:180]: or name mangling python voodoo?

**Review by jwilliams-ocient — APPROVED** [review]
Thanks!

File: superset/db_engine_specs/ocient.py (hunk 1 above)
Comment (alexclavel-ocient) [reply superset/db_engine_specs/ocient.py:180]: yes, __ is the frontend call for `flask_babel`

Comment (alexclavel-ocient) [reply superset/db_engine_specs/ocient.py:180]: I don't like it, but that is how all of the dialects do it

**Pull request merged**